StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/CheckmarkCircleView.swift

```diff
@@ -1,53 +0,0 @@
-//
-//  CheckmarkCircleView.swift
-//  StripePaymentSheet
-//
-//  Created by Nick Porter on 5/9/24.
-//
-
-import Foundation
-import UIKit
-
-/// Draws a circle with the desired fill color with a white checkmark in the center
-final class CheckmarkCircleView: UIView {
```

Comment:
Apologies for deleting just reviewed code, I stumbled upon the `Image.icon_checkmark` which made using `CircularButton` viable. Probably matches the mocks a little better too.